### ISSUE TITLE
paddledetection模型

### ISSUE THREAD

**Issue by lhy823436493** [PaddlePaddle/InterpretDL#10]
训练好了paddledetection模型，怎么使用interpretDL进行可视化？

**Comment by holyseven** [comment]
暂时没有直接支持detection模型的实现。

但可以参考 Issue #5 ，对mask RCNN进行可视化分析。

**Comment by lhy823436493** [comment]
虽然是检测，我只是想输出backbone部分的热力图，使用的yolo算法，backbone会输出三个output，我怎么对这三层进行可视化，尝试之后报错维度不匹配，target = paddle.sum(out * labels_onehot, axis=1)，out和labels_onehot维度不匹配

**Comment by lhy823436493** [comment]
就算是采用分类模型，用的是rexnet,需要将最后的输出一层out =self.output(x).squeeze()改为out =self.output(x)才可以运行，但是运行出来吗没有热力图，如下，而且有一个警告RuntimeWarning: invalid value encountered in true_divide
  heatmap /= heatmap_max.reshape((bsz, ) + (1, ) * (heatmap.ndim - 1))，能请教一下是什么原因吗?
![L3BOFP4YV`T1BDTIFURR~@X](https://user-images.githubusercontent.com/53975909/117331875-c1527c00-aec9-11eb-8e17-631db3a3538c.png)

**Comment by lhy823436493** [comment]
mobilenetv2网络也报错，是不是Gard-cam的底层代码有问题？对输出格式有强制性要求？
![9P)KS{F5{374`~U(5MTPKLY](https://user-images.githubusercontent.com/53975909/117401386-124c8980-af37-11eb-9640-17aebaeda945.png)

**Comment by holyseven** [comment]
This should work.

```python
import paddle
import interpretdl as it
from paddle.vision.models import mobilenet_v2, resnet50

paddle_model = mobilenet_v2(pretrained=True)
paddle_model.eval()

gradcam = it.GradCAMInterpreter(paddle_model, use_cuda=False)

img_path = 'assets/catdog.png'

heatmap = gradcam.interpret(
        img_path,
        'features.18.2',
        labels=None,
        visual=True,
        save_path=None)

```

**Comment by lhy823436493** [comment]
好的，谢谢您！
